tests/framework/Samplers/Restart/test_restart_Grid_part2.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" ?>
+<Simulation verbosity="debug">
+  <RunInfo>
+    <WorkingDir>gridSplit</WorkingDir>
+    <Sequence>load_old, makeCoarse,print</Sequence>
+    <batchSize>1</batchSize>
+  </RunInfo>
+  <TestInfo>
+    <name>framework/Samplers/Restart.Grid</name>
+    <author>cogljj</author>
+    <created>2023-01-06</created>
+    <classesTested>Samplers.Grid</classesTested>
+    <description>
+      This loads the data from part1, and then generates more data
+      and saves it. This is an example of how to write a new input
+      that continues a 'failed' run of RAVEN.
+    </description>
+  </TestInfo>
+
+  <Files>
+    <Input name="old_solns" type="">coarse.csv</Input>
+  </Files>
+
+  <Steps>
+    <IOStep name="load_old">
+      <Input class='Files' type=''>old_solns</Input>
+      <Output class="DataObjects" type="PointSet">solns_orig</Output>
+    </IOStep>
+    <MultiRun name="makeCoarse">
+      <Input class="DataObjects" type="PointSet">dummyIN</Input>
+      <Model class="Models" type="ExternalModel">coarsemod</Model>
+      <Sampler class="Samplers" type="Grid">coarse</Sampler>
+      <Output class="DataObjects" type="PointSet">solns</Output>
+    </MultiRun>
+    <IOStep name="print">
+      <Input class="DataObjects" type="PointSet">solns</Input>
+      <Output class="OutStreams" type="Print">coarse2</Output>
+    </IOStep>
+  </Steps>
+
+  <Distributions>
+    <Uniform name="u1">
+      <lowerBound>1</lowerBound>
+      <upperBound>2</upperBound>
+    </Uniform>
+    <Uniform name="u2">
+      <lowerBound>2</lowerBound>
+      <upperBound>3</upperBound>
+    </Uniform>
+  </Distributions>
+
+  <Samplers>
+    <Grid name="coarse">
+      <variable name="x1">
+        <distribution>u1</distribution>
+        <grid construction="equal" steps="2" type="CDF">0.0 1.0</grid>
+      </variable>
+      <variable name="x2">
+        <distribution>u2</distribution>
+        <grid construction="equal" steps="2" type="CDF">0.0 1.0</grid>
+      </variable>
+      <Restart class="DataObjects" type="PointSet">solns_orig</Restart>
+    </Grid>
+  </Samplers>
+
+  <Models>
+    <Dummy name="MyDummy" subType=""/>
+    <ExternalModel ModuleToLoad="../fine" name="coarsemod" subType="">
+      <variables>x1,x2,ans</variables>
```

Comment:
Could you use 'inputs' and 'outputs' instead of 'variables'?

Reply:
Fixed.